composer.json

```
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 | ^8.0",
```

Pretty sure the syntax for a logical `OR` is a double pipe. See https://getcomposer.org/doc/articles/versions.md#version-range

> You can define multiple ranges. Ranges separated by a space ( ) or comma (,) will be treated as a logical AND. A double pipe (||) will be treated as a logical OR. AND has higher precedence than OR.

```suggestion
        "php": "^7.2 || ^8.0",
```